### PR TITLE
[lab-supervised-learning-sklearn] Ana Horta

### PR DESCRIPTION
**Pull request by japana26**

### CODE REVIEW

**Comment by ta-data-lis** [comment]
Lab checked. Well done! ✨

The point of splitting the data into train and test sets is to keep some data on reserve (the test set) that you do not show to the model until after it's been trained (on the train set). This way, you can better evaluate its performance (that is, how good are the results it produces when it encounters all-new data). So, when you first fit the model, you will only want to include the training data set, like so:
```
auto_model = LinearRegression().fit(X_train, y_train)
```

Keep up the good work! 💪
